pinion_adverts.sp

```diff
@@ -577,40 +607,57 @@ public OnPluginStart()
 #endif
 }
 
+public Action:FuncChatHook(Handle:event, const String:name[], bool:dontBroadcast)
+{
+	new client = GetClientOfUserId(GetEventInt(event, "userid"));
+	decl String:strChat[256]; 
+	GetEventString(event, "text", strChat, sizeof(strChat));
+
+	if (StrContains(strChat, "!RewardMe", false) == 0)
```

Comment:
Yes, per @sirmoe.  That's why a chat hook was implemented rather than using RegConsoleCmd.